src/replication/ng/model/execution.go

```diff
@@ -37,7 +58,19 @@ type Task struct {
 	SrcResource  string       `json:"src_resource"`
 	DstResource  string       `json:"dst_resource"`
 	JobID        string       `json:"job_id"`
-	Status       string       `json:"status"`
+	Status       TaskStatus   `json:"status"`
```

Comment:
define a type TaskStatus brings translation to dao.model and mananger.model and makes model could not be shared in dao and manager even they looks the same.